environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/EnvironmentStatus.java

```diff
@@ -39,6 +39,7 @@
 
     ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_IN_PROGRESS("Initializing Resource Encryption settings for environment"),
     ENVIRONMENT_RESOURCE_ENCRYPTION_DELETE_IN_PROGRESS("Deleting Resource Encryption settings of environment"),
+    ENVIRONMENT_ENCRYPTION_RESOURCES_DELETED("Deleted Resource Encryption settings for environment"),
```

Comment:
ui change is required

Reply:
Updated CB-11791 to include this UI change.